packages/salesforcedx-vscode-lwc/test/vscode-integration/commands/forceLightningLwcPreview.test.ts

```diff
@@ -317,6 +318,7 @@ describe('forceLightningLwcPreview', () => {
       'streamCommandOutput'
     );
     appendLineSpy = sinon.spy(ChannelService.prototype, 'appendLine');
+    isSFDXContainerModeStub.returns(false);
```

Comment:
We can only stub functions within an object or class. Since it's a stand alone function, not able to stub it. I think we need to modify the implementation of isSFDXContainerMode()